data/json/recipes/recipe_others.json

```diff
@@ -1170,6 +1170,20 @@
     "tools": [ [ [ "welder", 50 ], [ "welder_crude", 75 ], [ "toolset", 75 ], [ "oxy_torch", 10 ] ] ],
     "components": [ [ [ "wire", 6 ], [ "basket", 1 ] ], [ [ "scrap", 8 ] ] ]
   },
+  {
+    "type": "recipe",
+    "result": "bike_basket",
+    "category": "CC_OTHER",
+    "subcategory": "CSC_OTHER_PARTS",
+    "skill_used": "fabrication",
+    "difficulty": 1,
+    "time": 5000,
+    "reversible": true,
+    "autolearn": true,
+    "qualities": [ { "id": "GLARE", "level": 2 } ],
+    "tools": [ [ [ "welder", 20 ], [ "welder_crude", 35 ], [ "toolset", 35 ], [ "oxy_torch", 5 ] ] ],
```

Comment:
Please use `"using": [ [ "welding_standard", 2 ] ],` instead of list of welding tools and `GLARE` quality.

`welding_standard` includes all necessary qualities and tools: 

```
  {
    "id": "welding_standard",
    "type": "requirement",
    "//": "Crafting or repair of steel items or installation of vehicle parts",
    "qualities": [ { "id": "GLARE", "level": 2 } ],
    "tools": [ [ [ "welder", 10 ], [ "welder_crude", 15 ], [ "toolset", 15 ], [ "oxy_torch", 2 ] ] ]
  },
```